hadoop-ozone/dist/src/shell/ozone/ozone

```diff
@@ -121,6 +121,7 @@ function ozonecmd_case
       ozone_deprecate_envvar HDDS_DN_OPTS OZONE_DATANODE_OPTS
       OZONE_DATANODE_OPTS="${RATIS_OPTS} ${OZONE_DATANODE_OPTS}"
       OZONE_DATANODE_OPTS="-Dlog4j.configurationFile=${OZONE_CONF_DIR}/dn-audit-log4j2.properties ${OZONE_DATANODE_OPTS}"
+      OZONE_DATANODE_OPTS="-Dlog4j.configurationFile=${OZONE_CONF_DIR}/dn-container-log4j2.properties ${OZONE_DATANODE_OPTS}"
```

Comment:
It will take first or last occurance of configuration, but do not merge both configuration. IMO, it should be comma separated list. Also check if both logs are commit at same point of time, like merge, instead of overwrite for global configuration

Reply:
Right, it should be a comma separated list as specified [here](https://logging.apache.org/log4j/2.x/manual/configuration.html#SystemProperties). I updated the code.

> Also check if both logs are commit at same point of time, like merge, instead of overwrite for global configuration

I'm not sure what is meant by this comment, can you explain a bit more?

Reply:
I have renamed the properties in the container log4j configuration according to the [docs](https://logging.apache.org/log4j/2.x/manual/configuration.html#CompositeConfiguration) so that they will be merged instead of overwritten. After this change I can see that both the audit log and container log file are being created inside the datanode container.